include/can/core/messages.hpp

```diff
@@ -113,22 +113,18 @@ struct GetStatusResponse : BaseMessage<MessageId::get_status_response> {
 };
 
 struct MoveRequest : BaseMessage<MessageId::move_request> {
-    uint32_t distance;
-    uint32_t speed;
+    uint32_t steps;
```

Comment:
Hold on here. I thought we wanted the can message to contain speed and distance?

Reply:
I changed it for the demo. If we want to keep it as we expect it to be in the future, that's fine but we'll need to use UART for the demo then.

Reply:
Understood.

You'll have to fix some unit tests I believe.

Reply:
yep I'm in the midst of doing that now.